generated/elasticsearch/README.md

```diff
@@ -8,35 +8,162 @@ please check out [USAGE.md](../../USAGE.md).
 
 ## Notes on index naming
 
-This sample Elasticsearch template will apply to any index named `try-ecs-*`.
+These sample Elasticsearch templates will apply to any index named `try-ecs-*`.
 This is good for experimentation.
 
 Note that an index following ECS can be named however you need. There's no requirement
 to have "ecs" in the index name.
 
 ## Instructions
 
+Elasticsearch 7.8 introduced
+[composable index templates](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-templates.html)
+as the new default way to craft index templates.
+
+The following instructions let you use either approach.
+
+### Composable index templates
+
+**Warning**: The artifacts based on coposable templates are newly introduced in the ECS repository.
+Please try them out and give us feedback if you encounter any issues.
+
+If you want to play with a specific version of ECS, check out the proper branch first.
+Note that the composable index templates are available in the ECS 1.7 branch or newer.
```

Comment:
The mention of the 1.7 branch here was intentional. I figured 1.8 will be out in the new year, and thought it would be worthwhile to backport all the way to 1.7 for folks who are already using composable templates. (cc @a03nikki and friends in consulting)

The composable templates are not tied to any change in the schema itself, so I don't expect any problems doing so. The backport itself should be straightforward.

If you see a reason why we'd want to hold off and just release this in 1.8, I'm happy to target 1.8 instead.